src/process/functionality/image/create.js

```diff
@@ -0,0 +1,17 @@
+const Amplify = require('aws-amplify')
+const serializeAxiosResponse = require('../../../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../../../utils/defaultReqOpts')
+/**
+ * Uploads a process image, the image needs to be encoded in base64
+ * requires the image to be encoded in base64
+ * and the file type as string
+ * @param {string} ProcessId @param {string} FunctionalityID @param {object} Body
```

Comment:
```suggestion
 * @param {string} ProcessId @param {string} FunctionalityId @param {object} Body
```

(or big D on processId)